components/sup/doc/http_gateway_census_schema.json

```diff
@@ -0,0 +1,316 @@
+                },
+                "election_is_running": {
+                  "type": "boolean",
+                  "description": "Whether there is a leader election running for this service"
+                },
+                "environment": {
+                  "type": "null",
+                  "description": "The environment portion of a service group specification",
+                  "additionalProperties": {
+                    "type": ["null", "boolean", "string", "integer", "array", "object"]
+                  }
+                },
+                "follower": {
+                  "type": "boolean",
+                  "description": "Whether this member is a follower"
+                },
+                "group": {
+                  "type": "string",
+                  "description": "The group portion of a service's complete group name"
+                },
+                "leader": {
+                  "type": "boolean",
+                  "description": "Whether this member is a leader"
+                },
+                "member_id": {
+                  "type": "string",
+                  "description": "This member's ID"
+                },
+                "org": {
+                  "type": ["null", "string"],
+                  "description": "The organization of a service group specification"
+                },
+                "persistent": {
+                  "type": "boolean",
+                  "description": "Is this member a permanent peer",
+                },
+                "pkg": {
+                  "type": "object",
+                  "description": "The package identifier this member is running",
+                  "required": [
+                    "name",
+                    "origin",
+                    "release",
+                    "version"
+                  ],
+                  "properties": {
+                    "name": {
+                      "type": "string",
+                      "description": "The package name"
+                    },
+                    "origin": {
+                      "type": "string",
+                      "description": "The package origin"
+                    },
+                    "release": {
+                      "type": "string",
+                      "description": "The package release"
+                    },
+                    "version": {
+                      "type": "string",
+                      "description": "The package version"
+                    }
+                  }
+                },
+                "service": {
+                  "type": "string",
+                  "description": "The name of the service"
+                },
+                "suspect": {
+                  "type": "boolean",
+                  "description": "Whether this member is considered possibly unreachable, from a network perspective"
+                },
+                "sys": {
```

Comment:
I wonder if there's a straightforward way for us to share type definitions across our schema files. For instance, I think this is essentially the same as https://github.com/habitat-sh/habitat/blob/d68309486b02dca9491ed4d9933e40e835631a20/components/sup/doc/render_context_schema.json#L219-L280

Of course, that raises the legitimate question of whether or not we truly want these structures to be the same between the HTTP gateway and the rendering context (maybe? :man_shrugging:)

In any event, it seems there are similar structures shared between the various HTTP gateway endpoints, so ensuring all those (at the very least) are in-sync with the same schema definitions would be nice.